src/utils/authentication.js

```diff
@@ -0,0 +1,20 @@
+import jwt from 'jsonwebtoken';
+
+export const generateToken = (payload, expiresIn = '24h') => jwt.sign(payload, process.env.JWT_SECRET, { expiresIn });
+
+export const shuffleToken = (token) => {
+  token.split('').reverse().join('');
+};
+
+export const verifyToken = (req, res, input) => {
+  const token = shuffleToken(input);
+  jwt.verify(token, process.env.JWT_SECRET, (err, decoded) => {
+    if (err) {
+      return res.status(401).json({
```

Comment:
Instead of calling `res.status` here, if the token is invalid return an object with the `success` property set to false. and if the token is valid n verified return an object with property `success` set to true along with the payload of the token.